.github/tests/check_for_whitespace_before_comma.py

```diff
@@ -26,6 +26,6 @@ def find_bad_lines(file_content):
             sys.exit(1)
 
     except FileNotFoundError:
-        print(f"Die Datei '{file_path}' wurde nicht gefunden.")
+        print(f"Die Datei \"{file_path}\" wurde nicht gefunden.")
```

Comment:
Ist das eine Art "Best Practice" die ich noch nicht kenne?

Reply:
Ne passt schon, ich persönlich finde nur \" hässlich :-)
Ich hätte es so gemacht:
```suggestion
        print(f'Die Datei "{file_path}" wurde nicht gefunden.')
```

Aber ist ja auch so wie es jzt ist in Ordnung :-D